include/mpsl/mpsl_assert.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/**
+ * @file
+ * @brief MPSL assert
+ */
+
+#ifndef MPSL_ASSERT__
+#define MPSL_ASSERT__
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#if IS_ENABLED(CONFIG_MPSL_ASSERT_HANDLER)
```

Comment:
ifdefs for header files shouldn't be used for function definitions, only to exclude struct members in order to save memory.

Otherwise the function will not be included in the documentation build.

We should probably have documentation for this function.

Reply:
updated